docs/wf_options/Google.md

```diff
@@ -19,6 +20,7 @@ Keys | Possible Values | Description
   "refresh_token": "1/Fjf8gfJr5fdfNf9dk26fdn23FDm4x",
   "google_compute_service_account": " my-new-svcacct@my-google-project.iam.gserviceaccount.com"
   "auth_bucket": "gs://my-auth-bucket/private",
-  "monitoring_script": "gs://bucket/script.sh"
+  "monitoring_script": "gs://bucket/script.sh",
+  "monitoring_image": "gcr.io/my_google_project/my-monitoring-image"
```

Comment:
should use `quay.io/broadinstitute/cromwell-monitor` as you suggest above

Reply:
Ok! If this is going to be the "official" url, we will need to 1) hand over control over that Quay repo to your team; 2) configure automatic builds in Quay triggered by the GitHub Cromwell repo (to build off of the monitoring Dockerfile in the Cromwell repo).

Alternatively, we could add a CI script to the Cromwell repo (though I'm not sure where and in which form).

Also, is everyone ok with the name (`cromwell-monitor`), or should it be more specific, like `cromwell-task-stackdriver-monitor` or similar?

Thanks!

Reply:
Perfect, thanks. Submitting the fix for the doc.